torchgeo/models/resnet.py

```diff
@@ -16,7 +16,51 @@
 __all__ = ["ResNet50_Weights", "ResNet18_Weights"]
 
 _zhu_xlab_transforms = AugmentationSequential(
-    K.Resize(256), K.CenterCrop(224), data_keys=["image"]
+    K.Resize(256),
+    K.CenterCrop(224),
+    K.Normalize(mean=0, std=10000),
+    data_keys=["image"],
+)
+
+# https://github.com/ServiceNow/seasonal-contrast/
+# blob/8285173ec205b64bc3e53b880344dd6c3f79fa7a/datasets/bigearthnet_dataset.py#L13
```

Comment:
I dislike breaking long URLs across multiple lines. You can usually add `# noqa: E501` to the end of the line to get flake8 to ignore it. I only do this in situations where readability is improved by having a long line. Doing the same thing to our weight URLs in my weight rename PR.